### PR TITLE
Feat:Respondent Journey setup

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-926

### Change description ###

Set up docker environment for manage orgs end to end

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: bin/add-roles.sh [hunk 1]
```diff
@@ -23,3 +23,28 @@ do
...
+
+${dir}/utils/idam-add-role.sh "caseworker-publiclaw"
+${dir}/utils/idam-add-role.sh "caseworker-publiclaw-solicitor"
+
+${dir}/utils/idam-add-role.sh "xui-approver-userdata"
+
+${dir}/utils/idam-add-role.sh "caseworker-divorce"
+${dir}/utils/idam-add-role.sh "caseworker-divorce-solicitor"
+${dir}/utils/idam-add-role.sh "caseworker-divorce-financialremedy"
+${dir}/utils/idam-add-role.sh "caseworker-divorce-financialremedy-solicitor"
```
Comment (pd-dudkiewicz) [inline bin/add-roles.sh:47]: are we going to use roles from other services?

File: bin/add-roles.sh [hunk 2]
```diff
@@ -23,3 +23,28 @@ do
...
+
+prdRoles=('"caseworker"','"caseworker-caa"','"caseworker-divorce"','"caseworker-divorce-solicitor"','"caseworker-divorce-financialremedy"','"caseworker-divorce-financialremedy-solicitor"','"caseworker-probate"','"caseworker-ia"','"caseworker-probate-solicitor"','"caseworker-publiclaw"','"caseworker-ia-legalrep-solicitor"','"caseworker-publiclaw-solicitor"','"caseworker-civil"','"caseworker-civil-solicitor"','"xui-approver-userdata"','"pui-caa"','"prd-admin"','"pui-case-manager"','"pui-finance-manager"','"pui-organisation-manager"','"pui-user-manager"')
```
Comment (pd-dudkiewicz) [inline bin/add-roles.sh:49]: this looks like duplicate for roles listed above

File: bin/users.json [hunk 3]
```diff
@@ -1,5 +1,8 @@
 [
-  {"email": "solicitor@example.com", "roles": "caseworker,caseworker-civil,caseworker-civil-solicitor", "lastName": "Solicitor"},
-  {"email": "claimantsolicitor@gmail.com", "roles": "caseworker,caseworker-civil,caseworker-civil-solicitor", "lastName": "Solicitor"},
-  {"email": "civil-system-update@mailnesia.com", "roles": "caseworker,caseworker-civil,caseworker-civil-systemupdate", "lastName": "System-update"}
+  {"email": "solicitor@example.com", "roles": "caseworker,caseworker-civil,caseworker-civil-solicitor,pui-caa,,pui-organisation-manager,pui-case-manager,pui-user-manager", "lastName": "Solicitor"},
```
Comment (pd-dudkiewicz) [inline bin/users.json:2]: there's double comma in there `pui-caa,,pui-organisation-manager`

File: bin/assign-case.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+fi
+
+dir=$(dirname ${0})
+
+DEFENDANT_EMAIL=${1}
+CASE_ID=${2}
+
+echo -e "\nAssigning case to defendant: ${DEFENDANT_EMAIL}"
+
+userToken=$(${dir}/utils/idam-access-token.sh "${DEFENDANT_EMAIL}" "Password12")
```
Comment (pd-dudkiewicz) [inline bin/assign-case.sh:16]: isn't it `Password12!` now?

File: bin/assign-case.sh [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+echo "${userToken}"
```
Comment (pd-dudkiewicz) [inline bin/assign-case.sh:18]: we probably don't need to print that

File: bin/utils/idam-access-token.sh [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -eu
+
+USERNAME=$1
+PASSWORD=$2
+
+curl --silent --show-error --header 'Content-Type: application/x-www-form-urlencoded' -H 'Accept: application/json' -d "username=${USERNAME}&password=${PASSWORD}" "${IDAM_API_BASE_URL:-http://localhost:5000}/loginUser" | docker run --rm --interactive stedolan/jq -r .access_token
```
Comment (pd-dudkiewicz) [inline bin/utils/idam-access-token.sh:8]: `bin/utils/idam-authenticate.sh` looks almost the same, could probably make it into one script with additional parameter

File: bin/add-roles.sh (hunk 2 above)
Comment (dharmendrak) [reply bin/add-roles.sh:49]: I have followed aat assigned roles for role `prd-admin`

**Review by pd-dudkiewicz — DISMISSED** [review]

**Review by pd-dudkiewicz — APPROVED** [review]

**Pull request merged**